dashboard/test/ui/runner.rb

```diff
@@ -25,6 +27,26 @@
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+S3_LOGS_BUCKET = 'cucumber-logs'
+S3_LOGS_PREFIX = GitUtils.current_branch
```

Comment:
I didn't consider the case where `./runner.rb` is being used to test a remote machine running a different branch than the local machine has checked out.

I thought about prefixing logs `<hostname>/<branchname>` to cover cases like this (local developer has staging checked out but is running against test machine, local developer has test checked out but is running against local changes, etc) so that only the test machine would be dropping stuff into the `test/test` prefix.

This would be easy to switch to, but based on our usual workflow I don't expect it to become an issue.  I also wasn't sure we wanted all of our machine hostnames in public log URLs.